docker/docker-dev-debug/metrictank.ini

```diff
@@ -0,0 +1,417 @@
+## misc ##
```

Comment:
this file is out of sync with `../../scripts/config/metrictank-docker.ini`. seems the only change we should have is the `prefix` setting

Reply:
Yes. I believe I may have initially copied the wrong file when I was putting this together, I will fix that.